windows/task.go

```diff
@@ -102,16 +104,38 @@ func (t *task) CloseIO(ctx context.Context) error {
 }
 
 func (t *task) Info() runtime.TaskInfo {
-	return runtime.TaskInfo{
+	taskInfo := runtime.TaskInfo{
 		ID:        t.id,
-		Runtime:   pluginID,
 		Namespace: t.namespace,
 		// TODO(mlaventure): what about Spec? I think this could be removed from the info, the id is enough since it matches the one from the container
 	}
+	if t.isWindows {
+		taskInfo.Runtime = windowsPluginID
+	} else {
+		taskInfo.Runtime = lcowPluginID
+	}
+	return taskInfo
 }
 
 func (t *task) Start(ctx context.Context) error {
-	conf := newWindowsProcessConfig(t.spec.Process, t.io)
+	var (
+		conf *hcsshim.ProcessConfig
+		err  error
+	)
+	if t.isWindows {
+		conf = newWindowsProcessConfig(t.spec.Process, t.io)
+	} else {
+		conf, err = newLinuxProcessConfig(t.spec.Process, t.io)
+		if err != nil {
+			return err
+		}
+		ociBuf, err := json.Marshal(t.spec)
```

Comment:
have this part be done by `newLinuxProcessConfig` too. will avoid duplicating the code for execs

Reply:
Actually, I double checked the platform, and this is only needed in the initial process, and not in exec. Adding a `newInitialLinuxProcessConfig` function that handles this, and leaving `newLinuxProcessConfig` as is to match the Windows counterpart.